data/Pirralho_2014/data.csv

```diff
@@ -1,4 +1,4 @@
-﻿Species,Vessel Tangential diameter (um),Vessel Length (um),Vessel Area (mm2),Vessel Frequency (nr/mm2),Rays N° cells,Rays Height (um),Fibre Length (mm),Fibre Width (um),Fibre Wall thickness (um),Runkel ratio,Wall proportion %,Flexibility coefficient,Slenderness ratio,Luce's factor,Vessels (%),Rays (%),Fibre (%),Parenchyma (%)
+Species,Vessel Tangential diameter (um),Vessel Length (um),Vessel Area (mm2),Vessel Frequency (nr/mm2),Rays N° cells,Rays Height (um),Fibre Length (mm),Fibre Width (um),Fibre Wall thickness (um),Runkel ratio,Wall proportion %,Flexibility coefficient,Slenderness ratio,Luce's factor,Vessels (%),Rays (%),Fibre (%),Parenchyma (%)
```

Comment:
What is the change here? It looks identical, but I assume some special character has changed?

Reply:
For some reason on my Github Desktop it showed that I removed a weird space character before 'Species'